src/ImageIO/Enums.cs

```diff
@@ -60,4 +61,14 @@ public enum CGImagePropertyPngFilters {
 		Average = 0x40,
 		Paeth = 0x80
 	}
+
+	[Mac (10, 15), iOS (13, 0), TV (13, 0), Watch (6, 0)]
+	public enum CGImageAnimationStatus
```

Comment:
where is that used ?

the animation API you added are also only on iOS and macOS, but the enum is marked as available on all platforms

Reply:
@spouliot  I think it should be the return type for both `AnimateImage` members please see my comment on them 😄

Reply:
I thought that the animation API is available on Watch and TV as well? It appears in the diffs for those device types as well as in the xtro output for all device types. Am I missing something? It's also marked as available across all in the Apple Docs, but I know we can't trust those...

Reply:
Apple headers can be confusing and are not consistent. Assume that if headers exists for a platform that:
* the API is available
* that it's available starting with this version of Xcode (and related SDK OS versions)
even if no availability macro for the platform is present in the headers.

Introspection, xtrospection and submission tests should be able to find mistakes.